qgisfeedproject/qgisfeed/views.py

```diff
@@ -268,12 +268,19 @@ def get(self, request):
         user_is_approver = user.has_perm("qgisfeed.publish_qgisfeedentry")
         form = self.form_class()
 
+        try:
```

Comment:
@Xpirix, could you please create a function for this? That way, you won't need to call it every time. Alos please create a test for the function.